### ISSUE TITLE
No tooltips for launcher entries (fix included)

### ISSUE THREAD

**Issue by FliegendeWurst** [eliandoran/breeze-trilium#6]
First of all, I have to say that this theme is simply amazing! It must have been a lot of work to mirror the Breeze style using only CSS / Less, all whilst working around the document structure provided by Trilium.

I only noticed one minor issue:

![image](https://github.com/eliandoran/breeze-trilium/assets/12560461/a11234cf-8890-4899-8764-3426871ddc67)

As you can see there is no tooltip to remind me of the launcher's function. It's not that important (I know all my launchers), but I have some with identical icons.

It is fixable with this bit of CSS, which will put the tooltip below the button (annoying side effect: even the buttons with labels get tooltips).
```css
.tooltip.bs-tooltip-right {
  position: absolute;
  left: calc(-4px + calc(-1 * var(--breeze-iconbutton-width))) !important;
  top: var(--breeze-iconbutton-width) !important;
}
```
Whilst removing this bit
https://github.com/eliandoran/breeze-trilium/blob/c426543d8ba63dfa40a961c31ee9fb5678d48834/src/toolbar.less#L180-L185

**Comment by eliandoran** [comment]
@FliegendeWurst ,

Firstly, I'm really glad to hear that this theme is being used by other people. I mostly did it for myself, but I do admit I spent quite some time and energy on it which is why i decided to publish it.

The proposed solution looks good, I've released it in https://github.com/eliandoran/breeze-trilium/releases/tag/v0.6.1, with some other minor fixes. Let me know if it's OK.

I think I opted to remove the tooltips entirely because there is currently no way to disable the tooltips for the ones having labels and it felt redundant for me. But it does indeed make more sense to keep them for the icons that are not self-explanatory. Maybe some time in the future I'll also add a JavaScript layer to do the final tweaks that can't be done in CSS alone.

**Comment by FliegendeWurst** [comment]
> Firstly, I'm really glad to hear that this theme is being used by other people. I mostly did it for myself, but I do admit I spent quite some time and energy on it which is why i decided to publish it.

I really like the theme because no other theme (that I could find) hits all of these:
- is compact
- has an unintrusive color scheme
- makes UI elements visibly separated

> The proposed solution looks good, I've released it in [v0.6.1 (release)](https://github.com/eliandoran/breeze-trilium/releases/tag/v0.6.1), with some other minor fixes. Let me know if it's OK.

Yep, it works well.
